### PR TITLE
add messages business rule delete button

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/410
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2330

### CODE REVIEW

File: BackofficeBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -122,6 +122,7 @@ open_orchestra_backoffice:
                 publication: Publication
                 data: Data
         content_type:
+            business_rules_help_text: The content type should be not used
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/translations/messages.en.yml]: The content type should be not be used

Comment (itkg-nanne) [inline BackofficeBundle/Resources/translations/messages.en.yml]: The content type should not be used

File: GroupBundle/Resources/translations/messages.en.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ open_orchestra_backoffice:
 open_orchestra_group:
     form:
         group:
+            business_rules_help_text: The group should be empty
```
Comment (itkg-nanne) [inline GroupBundle/Resources/translations/messages.en.yml]: The group should not contain members

File: GroupBundle/Resources/translations/messages.fr.yml [hunk 3]
```diff
@@ -3,6 +3,7 @@ open_orchestra_backoffice:
 open_orchestra_group:
     form:
         group:
+            business_rules_help_text: Le group doit être vide
```
Comment (itkg-nanne) [inline GroupBundle/Resources/translations/messages.fr.yml]: Le groupe ne doit pas contenir de membres

File: Backoffice/Form/Type/ContentType.php [hunk 4]
```diff
@@ -112,6 +114,8 @@ public function configureOptions(OptionsResolver $resolver)
             'is_statusable' => false,
             'need_link_to_site_defintion' => false,
             'delete_button' => false,
+            'delete_business_rules' => false,
```
Comment (itkg-nanne) [inline Backoffice/Form/Type/ContentType.php]: It sounds weird to bind an option with the button and the other with the manager. It should be show_delete_button and enable_delete_button or delete_right_rules and delete_business_rules.

**Comment by alavieille** [comment]
@itkg-nanne pr updated

**Comment by itkg-nanne** [comment]
:+1:

**Pull request merged**